src/components/fields/ObjectField.js

```diff
@@ -42,6 +69,13 @@ class ObjectField extends Component {
     readonly: false,
   };
 
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
it looks like we've got the class fields proposal enabled, so you can do this with:
```
state = {
   additionalProperties: {}, 
}
```